docs/nodes/validate/add-a-validator.md

```diff
@@ -48,182 +48,304 @@ The response has your node’s ID:
-        "stakeAmount": "2000000000000"
-      }
-    ]
-  },
-  "id": 1
+  "validators": [
+    {
+      "nodeID": "NodeID-7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg",
+      "startTime": "1654656829",
+      "endtime": "1662602029",
+      "stakeAmount": "1000000000"
+    }
+  ],
+  "delegators": []
 }
 ```
 
-When the time reaches `1584021450`, this node will start validating the Primary Network. When it reaches `1584121156`, this node will stop validating the Primary Network. The staked AVAX will be returned to an address controlled by `username`, and the rewards, if any, will be given to `rewardAddress`.
+When the time reaches `1654656829` (Wed Jun 08 2022 02:53:49 GMT+0000), this node will start validating the Primary Network. When it reaches `1662602029` (Thu Sep 08 2022 01:53:49 GMT+0000), this node will stop validating the Primary Network. The staked AVAX and the rewards, if any, will be returned to `pAddressStrings`.
+
+#### Customizing Addresses
+
+There are 4 addresses which are needed when calling `pchain.buildAddValidatorTx`. Only 2 of them can be changed: `toAddresses` and `rewardAddresses`. For backward-compatibility reasons, `fromAddresses` and `changeAddresses`
+are just placeholders and are ignored.
+
+`toAddresses`
+
+An array of addresses who receive the staked tokens at the end of the staking period.
+
+`rewardAddresses`
+
+When a validator stops validating the Primary Network, they will receive a reward if they are sufficiently responsive and correct while they validated the Primary Network. These tokens are sent to `rewardAddresses`. The original stake will be sent back to the addresses defined in `toAddresses`.
+
+A validator’s stake is never slashed, regardless of their behavior they will always receive their stake back when they’re done validating.
+
+Locate this part of the code
+
+```ts
+let privKey: string = `${PrivateKeyPrefix}${DefaultLocalGenesisPrivateKey}`
+pKeychain.importKey(privKey)
+```
+
+and replace `privKey` with [private keys that you control](https://github.com/ava-labs/avalanchejs/blob/master/examples/platformvm/createKeypair.ts).
+
+```ts
+let privKey: string = `${PrivateKeyPrefix}${DefaultLocalGenesisPrivateKey}`
+pKeychain.importKey(privKey)
+privKey = "PrivateKey-FLKnwy3eFr75LHw6mPn8RAsAcWPq4WD8erMWiTK4KsLvkuP2t"
+pKeychain.importKey(privKey)
+privKey = "PrivateKey-2AUhohZSb86GKXE3mTBNVQGDWYpgR3FGHh7r6gqFoUzXDBNCcA"
+pKeychain.importKey(privKey)
+priKey = "PrivateKey-JeiGENXTFw5NWzBR18iacf1mL18VosfHf5eURhvmqSiAZKLaC"
+pKeychain.importKey(privKey)
+
+const pAddressStrings: string[] = pchain.keyChain().getAddressStrings()
+```
+
+This example would create a keychain with 4 addresses:
+
+```ts
+[
+  "P-custom18jma8ppw3nhx5r4ap8clazz0dps7rv5u9xde7p", // pAddressStrings[0]
+  "P-custom186e8ee4qnhnazrnyv2k9u44607ux70syngdz6l", // pAddressStrings[1]
+  "P-custom1pdu8d8ss5p329l4vtn23p25cjr57qt5yw2cnsn", // pAddressStrings[2]
+  "P-custom152s2upkznxeucwnp68wmg6wmy863qd2sz6ktww", // pAddressStrings[3]
```

Comment:
This is not the correct address for Fuji as this section is still under the Fuji Workflow. @Julian-dev28 please change the text how to generate the correct address value for Fuji.